### PR TITLE
Add e2e tests for hbar/token allowance

### LINKED ISSUE #1681: Add additional E2E tests for HBAR/Token allowances.
### Problem

We don't have full coverage of approvals/transfers and allowances for HBAR/Token when wrapped like ERC20.

### Solution

Add and expand current test cases so we can have more coverage and notice, if there are regressions.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by natanasow**
**Description**:

There was a test but was commented on because the image with the fix wasn't ready. HBAR approval could be done only via hapi but we don't test hapi calls in acceptance tests, so no actions are needed.

PS: charts install fails due to removing pnpm from dockerfile, so it has to be added in packages/server as a dependency

**Related issue(s)**:

Fixes #1681 

**Blocked by**:
- https://github.com/hashgraph/hedera-mirror-node/pull/6778

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by isavov — APPROVED** [review]

**Review by AlfredoG87 — APPROVED** [review]
LGTM

File: packages/server/package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
         "mocha": "^9.2.2",
         "pino": "^7.11.0",
         "pino-pretty": "^7.6.1",
-        "uuid": "^3.3.2"
+        "uuid": "^3.3.2",
+        "pnpm": "^8.6.6"
```
Comment (AlfredoG87) [inline packages/server/package.json:22]: just one question? 
why was pnpm needed here?

Comment (natanasow) [reply packages/server/package.json:22]: It's fixing https://github.com/hashgraph/hedera-json-rpc-relay/issues/1672. The `pnpm` was removed from the DockerFile and the `charts install` ci could not be able to build `packages/server` and that's why it started to fail recently. Adding `pnpm` as a dependency in `packages/server` will make `charts install` ci to succeed.

Comment (AlfredoG87) [reply packages/server/package.json:22]: Ohh, I see, thanks!! indeed the Test has passed on this branch. 👍 
Thank you!

**Review by Nana-EC — APPROVED** [review]

**Comment by AlfredoG87** [comment]
@natanasow should we split this PR in 2? so we can move forward with the fix of #1672 ?

**Comment by natanasow** [comment]
> @natanasow should we split this PR in 2? so we can move forward with the fix of #1672 ?

Yep, yesterday I found a bug in the latest mirror node image and this PR will be blocked for a while. Will open a separate PR only for #1672 fix :rocket:.

**Comment by Nana-EC** [comment]
@natanasow you should rebase. I think Eric address the other items so it should just be the  pnpm part

**Comment by natanasow** [comment]
> @natanasow you should rebase. I think Eric address the other items so it should just be the pnpm part

The `pnpm` had been already added in another PR I guess, so this PR could be closed because it doesn't change anything right now :confused: .

**Pull request closed without merging**